js/adapt-stateful-session.js

```diff
@@ -79,6 +79,24 @@ define([
 			this.listenTo(Adapt.blocks, 'change:_isComplete', this.onBlockComplete);
 			this.listenTo(Adapt.course, 'change:_isComplete', this.onCompletion);
 			this.listenTo(Adapt, 'assessment:complete', this.onAssessmentComplete);
+			this.listenTo(Adapt, 'app:resetSession', this.onResetSession);
```

Comment:
I had a hard time tracking down where this event gets triggered from. Eventually Ollie showed me it is in [Language Picker](https://github.com/adaptlearning/adapt-contrib-languagePicker/blob/master/js/languagePickerModel.js#L91).

Since we use Adapt's global events system, the prefix is normally used to signify its origin e.g. anything with `app:` I'd expect to be from core's `app.js`.

Suggest we rename this event to make it easier to track down in the future? `languagePicker:resetSession` seems the most logical.